rush-plugins/rush-migrate-subspace-plugin/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "rush-migrate-subspace-plugin",
+  "version": "1.0.0",
+  "description": "Rush plugin to migrate existing projects into a subspace",
+  "keywords": [
+    "rush",
+    "plugin",
+    "migrate",
+    "subspace"
+  ],
+  "homepage": "https://github.com/tiktok/rush-plugins#readme",
+  "bugs": {
+    "url": "https://github.com/tiktok/rush-plugins/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tiktok/rush-plugins.git",
+    "directory": "rush-plugins/rush-migrate-subspace-plugin"
+  },
+  "license": "MIT",
+  "author": "pedro.gomes",
+  "main": "lib/main.js",
+  "bin": {
+    "rush-migrate-subspace": "lib/cli.js"
+  },
+  "scripts": {
+    "build": "heft build --clean",
+    "prepublishOnly": "npm run build",
+    "start": "heft build-watch",
+    "test": "heft test"
+  },
+  "engines": {
+    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
+  },
+  "dependencies": {
+    "@rushstack/node-core-library": "3.62.0",
+    "@rushstack/rush-sdk": "^5.139.0",
+    "commander": "~9.4.0",
+    "ts-node": "11.0.0-beta.1"
```

Comment:
what's the usage of `ts-node` dependency? Is possible to remove this? It's relevant because it's a dependency rather than "devDependency", which means it will be installed in the user's end. It doesn't make sense to install useless dependencies in the user's end.

Reply:
You are right, also this is not being used, so i just removed it! thanks for letting me know